pkgs/by-name/ta/tacentview/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  libGL,
+  ninja,
+  stdenv,
+  tacent,
+  xorg,
+  zenity,
+  ...
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tacentview";
+  version = "1.0.46";
+
+  src = fetchFromGitHub {
+    owner = "bluescan";
+    repo = pname;
+    rev = version;
+    hash = "sha256-d4A26p1hmkYEZ+h6kRbHHr4QmAc3PMe3qYdkeKIRGkU=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+  ];
+
+  buildInputs = [
+    libGL
+    tacent
+    xorg.libX11
+    xorg.libxcb
+    zenity
+  ];
+
+  cmakeFlags = [
+    "-DFETCHCONTENT_SOURCE_DIR_TACENT=${tacent.src}"
+    "-DPACKAGE_NIX=True"
```

Comment:
```suggestion
    (lib.cmakeOption "FETCHCONTENT_SOURCE_DIR_TACENT" "${tacent.src}")
    (lib.cmakeBool "PACKAGE_NIX" true)
```

Reply:
Is this necessary? I didn't see any other packages using it.

Reply:
A simple `rg --type=nix cmakeBool` returns more than 400 results in `pkgs/by-name`.
Further, `lib.cmake*` functions type-checks their arguments.